packages/bierzo-wallet/src/routes/payment/components/CurrencyToSend/index.tsx

```diff
@@ -6,98 +6,89 @@ import Paper from '@material-ui/core/Paper';
 import Select from '@material-ui/core/Select';
 import TextField from '@material-ui/core/TextField';
 import { makeStyles } from '@material-ui/styles';
+import Block from 'medulas-react-components/lib/components/Block';
+import Typography from 'medulas-react-components/lib/components/Typography';
 import React from 'react';
 
 interface Props {
   readonly positionedClass?: string;
 }
 
 const useStyles = makeStyles((theme: Theme) => ({
-  paperRoot: {
+  paper: {
     display: 'flex',
     flexDirection: 'column',
     alignItems: 'center',
     width: '100%',
-    padding: '4rem',
+    padding: '40px',
   },
 
-  avatarRoot: {
+  avatar: {
     backgroundColor: '#ffe152',
-    fontSize: '2.75rem',
-    width: '7.2rem',
-    height: '7.2rem',
-    margin: '-7.6rem 0 4rem 0',
+    fontSize: '27.5px',
+    width: '72px',
+    height: '72px',
+    margin: '-76px 0 40px 0',
   },
 
   avatarIcon: {
     color: '#ffffff',
   },
 
-  sendLabel: {
-    color: theme.palette.text.primary,
-    fontFamily: '"Muli SemiBold", sans-serif',
-    fontSize: '1.4rem',
-  },
-
   currencyContainer: {
     display: 'flex',
```

Comment:
Where this prop is used, you should be able to use directly in the component: https://next.material-ui.com/system/flexbox/